src/main/java/com/revature/controllers/UsersByIdController.java

```diff
@@ -41,35 +41,34 @@
 
 	@Autowired
 	BamUserService userService;
-	
+
 	/**
 	 * Gets a specific user by their id
 	 * 
-	 * @author TJay Cargle 1801-jan8-java 
+	 * @author TJay Cargle 1801-jan8-java
 	 * 
 	 * @return User with that id
-	 * @param Integer userId of the user we want to grab
+	 * @param userId
+	 *            of the user we want to grab
 	 */
 	@GetMapping("/{userId}")
 	public BamUser getUsersById(@PathVariable Integer userId) {
 		BamUser user = userService.findUserById(userId);
 		return user;
 	}
-	
-	
+
 	/**
 	 * Updates the current user
-	 * @author Jeffrey Camacho (1712-Steve)
 	 * 
-	 * @LastUpdated John Brand and Joshua Stark (1802-Matt)
-	 * 		Moved to UsersByUserIdController
+	 * @author Jeffrey Camacho (1712-Steve)
+	 * @author John Brand (1802-Matt)
+	 * @author Joshua Stark (1802-Matt)
 	 * 
 	 * @return the updated BamUser
 	 * @param currentUser
 	 *            WE THINK THIS IS THE USER LOGGED IN, PLEASE CHANGE WHEN WORKED ON
 	 * @throws AuthUserException
 	 */
-	// TODO: FIND OUT WHAT CURRENTUSER IS
 	@PutMapping("/{userId}")
 	public BamUser updateUser(@RequestBody BamUser currentUser) throws AuthUserException {
 
```

Comment:
Manual set httpStatus code to 204?

Reply:
I looked online, they said 200 is fine for both of them, idk what the other people have though